pkg/common/keybindings.go

```diff
@@ -289,3 +306,295 @@ func matchesSequence(msg tea.KeyMsg, sequences []string) bool {
 	}
 	return false
 }
+
+// formatHelpKey formats a list of keys into a help string
+// Example: ["ctrl+c"] -> "^c", ["up", "k"] -> "↑/k", ["ctrl+alt+s"] -> "^⌥s"
+func formatHelpKey(keys []string) string {
```

Comment:
nice, kind of surprised this isn't baked into https://github.com/charmbracelet/bubbles